projects/uitdatabank/docs/entry-api/events/status.md

```diff
@@ -1,8 +1,141 @@
+
+Cancelled events automatically get an *"(geannuleerd)"* label next to their calendar info in online calendars. For example, on UiTinVlaanderen:
+
+<!-- focus: false -->
+
+![Screenshot of a summary of the event "Boshyacintenfestival" on UiTinVlaanderen, as an example of the "(Geannuleerd)" label](../../../assets/images/event-cancelled.png)
+
+*Note that when you cancel or postpone your event, you should **not** add AFGELAST or something similar in the title, as it will be redundant as can be seen above.*
+
+In this guide you will learn how to postpone and/or cancel your events, or specific dates of your events, via Entry API.
+
+Before getting started, we recommend that you have read the following guides:
+
+* [Creating a new event](./create.md) to learn about the basic structure of events
+* [Updating an event](./update.md) to postpone or cancel an event after creating it earlier
+* [Calendar info](../shared/calendar-info.md) for more info about the various calendar types and their properties
+
+## Permissions
+
+Anyone who [creates a new event](./create.md) can mark it as cancelled or postponed. For [updates to an existing event](./update.md), the usual permission checks apply.
+
+## The status property
+
+Every event automatically gets a `status` property after it is created.
+
+By default, it looks like this:
+
+```json
+{
+  "status": {
+    "type": "Available"
+  }
+}
+```
+
+The nested `type` property can either be `Available` (happening as scheduled), `Unavailable` (cancelled) or `TemporarilyUnavailable` (postponed). 
+
+The possible `type` values are intentionally worded in a generic way so that they match those of [the `status` property on places](../places/status.md), which makes it easier to query both events and places at the same time on their status property [using Search API](../../search-api/introduction.md).
+
+When the event has calendarType `single` or `multiple`, the objects inside its `subEvent` property will also automatically get the same `status` property. 
+
+For example on an event with multiple dates:
+
+```json
+{
+  "status": {
+    "type": "Available"
+  },
+  "calendarType": "multiple",
+  "subEvent": [
+    {
+      "startDate": "2023-01-18T17:30:00+01:00",
+      "endDate": "2023-01-18T23:00:00+01:00",
+      "status": {
+        "type": "Available"
+      }
+    },
+    {
+      "startDate": "2023-01-19T17:30:00+01:00",
+      "endDate": "2023-01-19T23:00:00+01:00",
+      "status": {
+        "type": "Available"
+      }
+    }
+  ]
+}
+```
+
+## Cancelling an event
+
+### With calendarType "single" or "multiple"
+
+To cancel an event with calendarType `single` or `multiple`, update the `status.type` property of the `subEvent` objects that should be cancelled and set it to `Unavailable`.
+
+Optionally, you can include a `reason` property to explain why the event is cancelled on that date. The `reason` property must be an object with language codes like `name`.
+
```

Comment:
The `reason` can only be combined with the statuses `Unavailable` and `temporarilyUnavailable`. When setting a (sub)Event back to `Available`, the reason(s) will be automatically removed.